includes/definitions/tgos.yaml

```diff
@@ -0,0 +1,13 @@
+os: tgos
+text: 'technicolor TG MediaAcces'
+type: network
+icon: technicolor
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.2863.405.1
+    - sysDescr_regex:
```

Comment:
You can remove this check. Using the sysObjectId is better and preferred.

Reply:
You speak about the "sysDescr_regex" part, right ?

Reply:
Correct (and obviously the line below it).